backend/connect.php

```diff
@@ -0,0 +1,47 @@
+**************************************************************************
+    Program establishes connection with account table within server to obtain
+    account_id of a specific user based on user credentials
+
+Current Version:
+    - obtains the one account within the database
+    - directs user to task creation 
+
+Next Version:
+    - add method of credential intake 
+
+    - add method of verification by matching credentials to entries on the table 
+        (i.e name and password searched within the table to obtain the 
+        corresponding account id)
+        (could include more unique identifiers like email, unique username, etc.)
+    
+**************************************************************************
+<?php
+    //connects to database
+    $conn = mysqli_connect('localhost', 'root', '9204Alex', 'coen146');
```

Comment:
Connection should call to variables set in a config file (do not leave these with your test credentials)